src/components/theme/App/App.jsx

```diff
@@ -62,20 +61,6 @@ class App extends Component {
     errorInfo: null,
   };
 
-  /**
-   * ComponentDidMount
-   * @method ComponentDidMount
-   * @param {string} error  The error
-   * @param {string} info The info
-   * @returns {undefined}
-   */
-  componentDidMount() {
```

Comment:
@zotya when we remove the old way of integrating Sentry we will break any project that relies on this or customized this code, right? Therefore we would need another major release if I am not mistaken. @plone/volto-team opinions?

Reply:
AFAIK the Sentry code was basically unusable in Volto: it only worked for the server, so I don't know if there's anybody using that or anything to break. The new code still reads the SENTRY_DSN env variable, it just moved its place where it's used.

Reply:
I can confirm it has been broken for a very long time. I think from the time we added the ``loadable`` library.